test/dlc_tests/ec2/mxnet/training/test_mxnet_training.py

```diff
@@ -18,6 +18,7 @@
 MX_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.4xlarge", processor="cpu")
 
 
+@pytest.mark.skip()
```

Comment:
Why is this test getting skipped ?

Reply:
Met some issues of building images. Try to trigger the build so modify on some test files